README.md

```diff
@@ -3,26 +3,26 @@
 </a>
     
 # cosima-recipes
-Example recipes using the cosima-cookbook infrastructure.
+Example recipes using the [`cosima-cookbook`](https://github.com/COSIMA/cosima-cookbook) infrastructure.
 
-To **Get Started** with the COSIMA Cookbook, clone this repository to your local file space.
-This will give you a bunch of examples with which you can begin to construct your own analysis code.
-If your code is useful, then **contributions back to the repository are always welcome**.
+To **Get Started** with the COSIMA Cookbook, clone this repository locally, probably best in your local space on one of the NCI HPC machines so you can have access to model output. The repository includes a bunch of examples with which you can begin to construct your own analysis code.
+
+If you develop a notebook for analysing something that is not already included in the recipes, then consider **contributing back to the repository via a Pull Requests**. Contributions are always welcome and we are more than happy to help out!
 
 ## [Tutorials](https://cosima-recipes.readthedocs.io/en/latest/tutorials/index.html)
 
-This directory contains a notebook (`COSIMA_CookBook_Tutorial.ipynb`) that outlines the basic philosophy of the COSIMA Cookbook. Best to start here if you can. Also some other tutorials regarding the cookbook (e.g., `Make_Your_Own_Database.ipynb`) or more general (`Making_Maps_with_Cartopy.ipynb`).
+The notebook `COSIMA_CookBook_Tutorial.ipynb` outlines the basic philosophy of the COSIMA Cookbook. This is the best place to start if you are not familiar with the `cosima-cookbook`. Also included here are some other tutorials, either related to the cookbook (e.g., `Make_Your_Own_Database.ipynb`) or more general (`Making_Maps_with_Cartopy.ipynb`).
```

Comment:
Just to note, this line refers to "COSIMA Cookbook", "`cosima-cookbook`" and "cookbook", and the line below also has `cosima_cookbook` — maybe some consistent branding throughout?

Reply:
Very good point 🤙🏼

Reply:
Agreed, applies throughout. I guess `cosima-cookbook` is best?

Reply:
resolved via 757a97f